docs/integrations/destinations/redshift-migrations.md

```diff
@@ -6,9 +6,21 @@ This version introduces [Destinations V2](/release_notes/upgrading_to_destinatio
 
 Worthy of specific mention, this version includes:
 
-- Per-record error handling
-- Clearer table structure
-- Removal of sub-tables for nested properties
-- Removal of SCD tables
+* Per-record error handling
+* Clearer table structure
+* Removal of sub-tables for nested properties
+* Removal of SCD tables
 
 Learn more about what's new in Destinations V2 [here](/using-airbyte/core-concepts/typing-deduping).
+
+## Upgrading to 3.0.0
```

Comment:
I think the QA check cares for the order with latest as top and 2.0.0 in the bottom.